inst/pages/acknowledgments.qmd

```diff
@@ -98,21 +98,21 @@ in writing the chapters on network learning and comparison.
 
 - *Ely Seraidarian*
 
--*Théotime Pralas*
+- *Théotime Pralas*
 
--*Muluh Muluh*
+- *Muluh Muluh*
 
--*Jiya Chaudhary*
+- *Jiya Chaudhary*
 
--*Elina Chiesa*
+- *Elina Chiesa*
 
--*Pande Erawijantari*
+- *Pande Erawijantari*
 
--*Shadman Ishraq*
+- *Shadman Ishraq*
 
--*Sam Hillman*
+- *Sam Hillman*
 
--*Matteo Calgaro*
+- *Matteo Calgaro*
 
```

Comment:
Could you check that the acknowledgment list contains all who have committed to mia, miaViz, miaSim, miaTime, OMA?

Reply:
Yes Done!